### PR TITLE
update(Files): Adjust max storage size and let free space automatically

### LINKED ISSUE #276: bug(files): user cannot upload files with large file size (example > 70 mb)
Tested on Safari

1. Create account
2. Go to Files
3. Attempt to upload a large file, I tried with one of 70 mb
4. A toast notification showing "An unknown error occurred" is displayed and file is not uploaded

### PR DESCRIPTION
**Pull request by lgmarchi**
### Which issue(s) this PR fixes 🔨

- Resolve #276

### CODE REVIEW

**Comment by luisecm** [comment]
Automated test failing on this PR since it is asserting the current free space/total space value. I have created a pr to fix the test and will merge once this pr is merged : https://github.com/Satellite-im/automated-tests-web/pull/99

**Review by luisecm — APPROVED** [review]

**Review by InfamousVague — APPROVED** [review]

**Pull request merged**
